### PR TITLE
monorepo setup, social previews server

### LINKED ISSUE #1904: Update docs to include monorepo and social media previews

### PR DESCRIPTION
**Pull request by rafalpawlow**
Closes #1904

### CODE REVIEW

**Review by kdembler — APPROVED** [review]
LGTM!

**Review by drillprop — APPROVED** [review]

**Review by eldiegod — APPROVED** [review]
looks good!

File: packages/atlas/.storybook/main.ts [hunk 1]
```diff
@@ -15,11 +19,7 @@ const config: CustomizedStorybookConfig = {
   'core': {
     'builder': 'storybook-builder-vite',
   },
-  viteFinal(storybookConfig, { configType }) {
-    if (configType === 'DEVELOPMENT') {
-      storybookConfig.build = {}
-    }
-    storybookConfig.build.target = ''
```
Comment (kdembler) [inline packages/atlas/.storybook/main.ts]: Why did you remove this?

File: packages/atlas/.prettierignore [hunk 2]
```diff
@@ -1,5 +1,13 @@
+node_modules/
 dist*/
+.yarn/
+storybook-static/
+.coverage
+tsconfig.json
+public/mockServiceWorker.js
```
Comment (kdembler) [inline packages/atlas/.prettierignore]: This is not needed

File: packages/atlas/.prettierignore [hunk 3]
```diff
@@ -1,5 +1,13 @@
+node_modules/
 dist*/
+.yarn/
+storybook-static/
```
Comment (kdembler) [inline packages/atlas/.prettierignore]: This is not needed

File: packages/atlas/.eslintignore [hunk 4]
```diff
@@ -1,4 +1,7 @@
-!.storybook/
-__generated__/
+node_modules
 dist*/
+storybook-static
```
Comment (kdembler) [inline packages/atlas/.eslintignore]: This is not needed

**Review by kdembler — CHANGES_REQUESTED** [review]

**Review by kdembler — APPROVED** [review]
Looks good! I will merge after all Giza migration is done, that should be tomorrow
